web/src/main/ui/app/components/flows-new/edit-flow/mapping/ui/mapping-ui.component.scss

```diff
@@ -45,32 +34,33 @@ p.item-identifying-info {
   font-weight: bold;
   margin: 0 0 14px 2px;
   font-size: 15px;
+  .sample-doc-uri {
+    padding-right: 7px;
+  }
 }
 
-.sample-doc-uri {
-  padding-right: 7px;
-}
-
-#source, #harmonized {
+#source, #target {
   position: relative;
   float: left;
   margin: 0;
 }
 
-#save-map {
-  position: relative;
-  margin: 0;
-}
-
 #source {
   width: 470px;
+  .src-prop-container {
```

Comment:
since we are moving on from "src", is it better to name this source-prop-container?

Reply:
@ayuwono, thanks, I fixed.